cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -186,6 +191,25 @@ pub mod ump_constants {
 	pub const MESSAGE_SIZE_FEE_BASE: FixedU128 = FixedU128::from_rational(1, 1000); // 0.001
 }
 
+/// Trait for selecting the next core to build the candidate for.
+pub trait SelectCore {
+	fn select_core_for_child() -> (CoreSelector, ClaimQueueOffset);
```

Comment:
so that it better expresses what this API returns. It doesn't return the core selector for the current block. It returns the core selector for the next block.